src/Connection.php

```diff
@@ -130,10 +131,10 @@ public function fetchAll(string $sql, array $bind = []): array
 
     public function fetchRoles(?string $roleLike = null): array
     {
-        $sql = 'SHOW ROLES';
+        $sql = 'SELECT * FROM INFORMATION_SCHEMA.APPLICABLE_ROLES';
```

Comment:
SHOW ROLES trva `462` sekund, a vypisuje aj role z inych warehouses, ... `SELECT` ... trva `17` sekund, a vypisuje role iba z daneho warehouse. Teda tu dava vacsi zmysel pouzit SELECT. Okrem toho sa v kode pouziva `CREATE ROLE IF NOT EXISTS` a toto sluzi iba na logovanie, ci uz rola existovala.

Vid diskusia na Slack: https://keboola.slack.com/archives/CQ1ASK06M/p1609918041022500